overcooked_ai_py/agents/agent.py

```diff
@@ -119,21 +120,22 @@ def joint_action(self, state):
         return joint_action_and_infos
 
 
-
 class AgentFromPolicy(Agent):
     """
     Defines an agent from a `state_policy` and `direct_policy` functions
     """
-
-    def __init__(self, state_policy, direct_policy, stochastic=True, action_probs=False):
+    
+    def __init__(self, state_policy, direct_policy, multi_state_policy, stochastic=True, action_probs=False):
         """
         state_policy (fn): a function that takes in an OvercookedState instance and returns corresponding actions
         direct_policy (fn): a function that takes in a preprocessed OvercookedState instances and returns actions
+        multi_state_policy (fn): a function that takes in multiple OvercookedState instatences and returns actions
```

Comment:
Minor typo here! instatences --> instances
(I'm not sure if I should flag this in PR review?!)

Reply:
(This multi_state_policy is a good idea by the way)

Reply:
Yeah, I've actually changed this API a tiny bit more, will be in a newer commit